src/modules/Bootloader.rb

```diff
@@ -166,6 +166,19 @@ def Read
 
         ::Bootloader::BootloaderFactory.current = ::Bootloader::BootloaderFactory.proposed
         ::Bootloader::BootloaderFactory.current.propose
+      rescue Errno::EACCES
+        # If the access to any needed file (e.g., grub.cfg when using GRUB bootloader) is not
+        # allowed, just abort the execution. Using Yast::Confirm.MustBeRoot early in the
+        # wizard/client is not enough since it allows continue.
+
+        Yast2::Popup.show(
+          # TRANSLATORS: warn to the user that is running the module without enough permissions
```

Comment:
Is really that useful to have a message to the translators to tell them what is already said in the string itself?

Something like `TRANSLATORS: pop-up message, beware the line breaks` would actually be useful, but the current one...

Reply:
Answer to the first question: actually no.

Thanks for the suggestion :)